src/section/section_keys.rs

```diff
@@ -76,7 +76,7 @@ impl SectionKeysProvider {
         &mut self,
         section_key_index: u64,
         our_name: &XorName,
-        elders_info: &EldersInfo,
+        _elders_info: &EldersInfo,
```

Comment:
Why not remove this parameter?

Reply:
this is covered/overwriten by your new fixing commit.
it will be removed after rebase/cherry-pick